sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -165,8 +168,12 @@ export interface QueueOptions {
   duplicateDetectionHistoryTimeWindow?: string;
 
   /**
-   * The URL of Service Bus queue to forward deadlettered messages to.
-   *
+   * Absolute URI or relative path of the messaging entity the dead-lettered messages are to be forwarded to.
```

Comment:
Provide an example for the absolute uri case.

Reply:
updated